.github/workflows/test-aws-auth.yml

```diff
@@ -0,0 +1,34 @@
+name: test-aws-auth
+
+on:
+  pull_request:
+    paths:
+      - 'aws-auth/**'
+      - '.github/workflows/test-aws-auth.yml'
+  push:
+    branches:
+      - main
+    paths:
+      - 'aws-auth/**'
+      - '.github/workflows/test-aws-auth.yml'
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./aws/auth
+        id: aws-auth
+        continue-on-error: true
```

Comment:
why?

Reply:
I was only interested in testing the output.

But if you prefer, I can turn this into an integration test that actually tries to authenticate

Reply:
all good